Packages/Willow-Tests.package/SingleSelectionWebViewTest.class/instance/testNotifyChangesWhenSelectionNotChanged.st

```diff
@@ -0,0 +1,16 @@
+tests-Single Selection-API
+testNotifyChangesWhenSelectionNotChanged
+
+	| singleSelectionView |
+
+	singleSelectionView := self newSingleSelectionView.
+	singleSelectionView allowAnyOf: #(1).
+	singleSelectionView notifyChangesTo: self.
+
+	notificationWasReceived := true.
+
+	singleSelectionView chooseAnySatisfying: [ :element | false ] ifNone: [ notificationWasReceived := false ].
+
+	self
+		deny: notificationWasReceived;
+		assert: (singleSelectionView currentSelectionIfNone: [ 0 ]) equals: 0
```

Comment:
should be renamed to testNotifyChangesWhenSelectionDoesNotChange
also, the last assert might benefit from using a specific string (such as "nothing found here") instead of a number, to ensure that the obtained result is not a classical return with 0.